conf/cb-event-forwarder.example.ini

```diff
@@ -152,6 +152,16 @@ syslogout=
 #   httpout=https://http-endpoint.company.local/api/submit
 httpout=
 
+# options for splunk output
+# splunkout:
+#   uses the format <temporary file location>:<splunk URL HEC interface URL:port>
+#   where the temporary file location is optional; defaults to /var/cb/data/event-forwarder
+#
+# for more http options, see the [splunk] section below.
+#
+# examples:
+#   splunkpout=localhost:8088/services/collector/events
```

Comment:
should the example read splunkout=http://splunkhost:8088/services/collector/events?
Also include an example with https to make it clear it works with https as well